src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -126,10 +114,11 @@ def generate_initial_population(self, pop_data: SimulantData) -> None:
         # drop non-unique household_id
         pop = pop.drop(columns="census_household_id")
 
-        # give names
-        first_and_middle = self.name_generator.generate_first_and_middle_names(pop)
-        last_names = self.name_generator.generate_last_names(pop)
-        pop = pd.concat([pop, first_and_middle, last_names], axis=1)
+        # give name ids
+        pop["first_name_id"] = pop.index
```

Comment:
So what is the plan here? At what point do these become actual id numbers?

Reply:
This is just going to be the id number.  We will handle in post-processing what these actually map to.  This takes away all generation and sampling for names out of the simulation.  So different ids may map to the same name.  We just want to make sure that if we know simulants will share a name that they will have the same name_id.  That is what the next ticket will implement for last name matching.